content/blog/2018/announcing-1.0/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Announcing Istio 1.0
+description: Istio 1.0
+publishdate: 2018-07-31
+weight: 84
+attribution: The Istio Team
+---
+
+Today, we’re excited to announce [Istio 1.0](/about/notes/1.0). It’s been a little over a year since our initial 0.1 release. Since then,
+Istio has evolved significantly with the help of a thriving and growing community of contributors and users. We’ve now reached the point
+where many companies have successfully adopted Istio in production and have gotten real value from the insight and control that it provides
+over their deployments. We’ve helped large enterprises and fast-moving startups like
+[[List off reference customers here]]
+use Istio to connect, manage and secure their services from the ground up. The 1.0 release has focused on platform and feature
+stability as well as improvements in performance. We have introduced a [performance regression framework]https://ibmcloud-perf.istio.io/regpatrol/)
+that conducts performance tests for each release looking for regressions in performance. Also, stay tuned for a more comprehensive Istio performance
+report due to be released soon. Shipping this release as 1.0 is recognition that we’ve built a stable set of functionality that our users can rely on
+for production use.
+
+We’ve also seen growth in the ecosystem around Istio. Observability providers like [Sysdig](https://sysdig.com/blog/monitor-istio/)], Datadog, and Solarwinds to
+name a few, have written plugins to integrate with their products. Cilium [[link]], Tigera & Styra
+[[link]] provided extensions to our networking and security capabilities. RedHat built [Kiali](https://www.kiali.io) to provide a nice
+user-experience around mesh management and observability.
+
+Our early adopters range from startups to large enterprises. The clients are using Istio to Our biggest motivation comes from the fact that Istio is helping
+them to solve complex microservices challenges with an ease. For us, Istio 1.0 is a recognition that our community has built a stable set of features that our
+users can rely on. We are continually amazed by the new set of features and integrations coming out of Istio.
+
+Here are few key features we launched in 1.0 release
+
+Since the 0.8 release we’ve added some important new features and more importantly marked many of our existing features as Beta signalling that they’re ready for production use. This is captured in more detail in the release notes [[link]] but here are some highlights:
+
+* Multi-cluster support allows for enrolling multiple Kubernetes clusters into a single mesh. This feature is now Beta.
+
+* Mutual TLS can now be rolled out incrementally without requiring all clients of a service to be updated. This is a critical feature that unblocks adoption for large existing production deployments.
+
+* Opaque TLS traffic can now be routed by host names specified in the SNI header.
+
```

Comment:
It would be good to capture things that are moving to stable and beta as part of 1.0 to show istio 1.0 getting production ready vs just the few things here.